libs/community/langchain_community/chat_models/tongyi.py

```diff
@@ -195,6 +196,8 @@ def convert_message_to_dict(message: BaseMessage) -> dict:
             message_dict["tool_calls"] = message.additional_kwargs["tool_calls"]
     elif isinstance(message, SystemMessage):
         message_dict = {"role": "system", "content": message.content}
+    elif isinstance(message, FunctionMessage):
+        message_dict = {"role": "assistant", "content": message.content}
```

Comment:
function message refers to the outputs of a function call — this isn't generated by the model. shoudl the role not be "function" or "tool", similar to the tool message below?